db/db.js

```diff
@@ -26,14 +27,6 @@ const getConnection = () => new Promise(async (resolve, reject) => {
   }).catch(err => reject(err));
 });
 
-// Sanitize raw data from database
-const sanitize = (row) => {
-  row.CAMPUS = row.CAMPUS ? row.CAMPUS.trim() : null;
-  row.CURRENT_ENROLLED = row.CURRENT_ENROLLED === 'Y';
-  row.CURRENT_REGISTERED = row.CURRENT_REGISTERED === 'Y';
-  return row;
-};
-
```

Comment:
This change is mainly to fix the following issue:

```
(node:34) UnhandledPromiseRejectionWarning: Error: ORA-12154: TNS:could not resolve the connect identifier specified
(node:34) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:34) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

This error reminds me that I put this `sanitize` helper function in the same file with other main database functions (e.g. create pools, get connections and execute queries, etc.), which causes the issue that the API will try to connect to the database every time even just running the unit tests, and this is not the behavior we want.